presentation/src/main/java/me/androidbox/presentation/edit/viewmodel/EditScreenViewModel.kt

```diff
@@ -27,7 +27,8 @@ class EditScreenViewModel @Inject constructor() : ViewModel() {
             }
             EditScreenEvent.OnSaveClicked -> {
                 Log.d("EDIT_SCREEN", "Content [ ${editScreenState.value} ]")
-                /* TODO Save the content which could be either the title of the description */
+                /* TODO Not sure about this - Should this be saved directly to the DB which could be either the title of the description
+                *   Or do we navigate back and save the title and description on the Edit Screen when the save is clicked */
```

Comment:
I am not sure about this part. If the save the title or description on this screen I would need the event ID to insert into the DB.

Reply:
This is the Edit Title and Edit Description?
Then I would navigate back with the new text

Why?
Because if you edit the db item, but then cancel the item edit (you don't click save, rather you click on close), then you updated a record in the DB when it wasn't really necessary